client/galaxy/scripts/mvc/ui/ui-misc.js

```diff
@@ -78,6 +78,57 @@ define(['utils/utils',
         }
     });
 
+
+    	var Password = Backbone.View.extend({
```

Comment:
Yes. It changes here: https://github.com/bgruening/galaxy/blob/b483138272747b754ae56aa36f53d775396cc982/client/galaxy/scripts/mvc/ui/ui-misc.js#L114. This seems to be the only line which needs differes from the regular 'text' input field.